package.json

```diff
@@ -1,8 +1,9 @@
 {
   "dependencies": {
-    "@rails/webpacker": "~4.0.7",
+    "@rails/webpacker": "^4.0",
```

Comment:
why was this relaxed?

Reply:
This is what we have on mcfe, so I'd guess this is an automatic update by npm. Note this Q on Stackoverflow:

> After I upgraded to latest stable node and npm, I tried npm install moment --save. It saves the entry in the package.json with the caret ^ prefix. Previously, it was a tilde ~ prefix. 

https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json

Reply:
Sorry my mistake. Have reverted